deploy/helm/sumologic/values.yaml

```diff
@@ -4348,7 +4348,7 @@ otellogs:
           - id: parser-crio
             type: regex_parser
             regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) (?P<log>.*)$'
-            output: extract-metadata-from-filepath
+            output: merge-cri-logs
```

Comment:
Just as a refresher: Why the `output` is set to a name of another operator?

Reply:
by default `output` is set to next operator in list, but we want to go to `merge-cri-logs` as next in pipeline

Reply:
That's how operators for filelog receiver work. You explicitly set which operator the output should go to if it's not the next one in the list.